LinaTheExplorer/LinaTheExplorer/Program.cs

```diff
@@ -1,12 +1,59 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
 
 namespace LinaTheExplorer
 {
     class Program
     {
         static void Main(string[] args)
         {
+            //var argsObj = new ArgsParser(args);
+            //List<int> tripDates = argsObj.getTripDates();
+            //string tripCity = argsObj.getTripCity();
+
+
+            #region localData
+            string tripCity = "Kyiv";
+            DateTime startVacFullDate = DateTime.Parse("2022/08/22");
+            int startVacDay = startVacFullDate.Day;
+
+            int countDays = 4;
+            var endVacFullDate = startVacFullDate.AddDays(countDays);
+            int endVacDay = endVacFullDate.Day;
+
+            var tripDates = new List<int>();
+
+            for (int day = startVacDay; day <= endVacDay; day++)
+            {
+                tripDates.Add(day);
+            }
+
+            #endregion
+
+            var listOfClothing = ClothingEntry.CreateListOfClothing();
             
+
+            List<WeatherData> weatherInfo = new List<WeatherData>();
+            int[] arrayOfTemp = WeatherResponse.GetWeather(tripCity);
+
+            foreach (var day in tripDates)
```

Comment:
building `tripDates` is redundant. Since you have the first and last month days of a vacation, you could iterate here as:
`for day = firstDay; day <= lastDay; day++`